pulsar-metadata/src/main/java/org/apache/pulsar/metadata/bookkeeper/AbstractMetadataDriver.java

```diff
@@ -67,6 +67,13 @@ protected void initialize(AbstractConfiguration conf) throws MetadataException {
         }
     }
 
+    public RegistrationManager createRegistrationManager() {
+        if (registrationManager == null) {
+            registrationManager = new PulsarRegistrationManager(store, ledgersRootPath, conf);
+        }
+        return registrationManager;
+    }
+
```

Comment:
Why do we make this change? It seems we generate an unreleased resource here.

cc @dlg99